plugins/modules/vlan.py

```diff
@@ -75,13 +75,19 @@
     required: false
     type: str
     version_added: "3.0.0"
+  group:
+    description:
+      - The group of the VLAN.
+    required: false
+    type: raw
+    version_added: "3.0.0"
```

Comment:
```suggestion
    version_added: "5.10.0"
```

Let's be true about the version it got added since it was only on the example up until now. Also, please add a changelog fragment for this as well.